.markdownlint.yaml

```diff
@@ -0,0 +1,36 @@
+# Enable all rules
+default: true
+
+# Enforce asterisk for unordered lists
+# See: https://github.com/DavidAnson/markdownlint/blob/main/doc/md004.md
+MD004:
+  style: asterisk
+
+# Set list indent level to 4 which Python-Markdown requires
```

Comment:
cf. "Indentation/Tab Length" [here](https://python-markdown.github.io/#differences)

I maintain dozen of mkdocs site, the indent 2 vs 4 is a recurring issue because indent of 2 only works for some parts, but you will never manage to make it work with:
- adminitions (indent hardcoded to 4)
- nested list including blocks

Reply:
Huh; worked fine for admonitions.

However, now I see that my assumption at the time (" must match indent") was mistaken. 🤯